sdl3-sys/Cargo.toml

```diff
@@ -27,6 +27,10 @@ link-static = []
 # Link SDL as a mac framework. The link-static feature has no effect if this is enabled.
 link-framework = []
 
+# When building from source, use the `SDL_UNIX_CONSOLE_BUILD` flag to disable the X11 dependency
+# and build a console-only version of SDL. This has no effect when not building from source.
+console-build = []
```

Comment:
Can you please rename this feature to `sdl-unix-console-build`? Thanks!

Reply:
Seems like I did the change while you were doing the review. Cheers!